### DISCUSSION TITLE
anyone know what happened to npcs run into player?

### DISCUSSION THREAD

**Discussion by infinity-stack** [Deadlineem/Extras-Addon-for-YimMenu#118] [General]
I think they removed it because it kept crashing but I kinda loved it cause it made people confused either they thought it was rockstar mods playing with them

**Comment by Deadlineem** [comment]
Unfortunately, this feature kept causing crashes in various forms, so until we find a permanent fix, like the vehicle spawner, its been removed.

**Comment by infinity-stack** [comment]
yeouch!
